### DISCUSSION TITLE
Journal Artile using Quarto in R Studio

### DISCUSSION THREAD

**Discussion by David-P-Moore** [quarto-dev/quarto-cli#4427] [Q&A]
Hi,

Would it be possible to assist with using journal template outputs in Quarto, using R Studio? I have recently migrated across from R Studio and the neat interplay between the rticles package and R Markdown, to using Quarto and rendering PDF documents to journal format is not quite as intuitive.

Sincerely,

David Moore.

**Comment by fortunewalla** [comment]
I am not the right person for this but it would be helpful if you added information about your installation and package versions and steps you are doing right now in Quarto to get your work done. 👍🏼

**Comment by mcanouil** [comment]
What do you mean by "assist"?

Did you take a look at <https://quarto.org/docs/journals/> and the few journal extension listed there? (few more are <https://quarto.org/docs/extensions/listing-journals.html>).

**Comment by David-P-Moore** [comment]
Thanks, Mickaël

I have visited those web pages, but unable to get my Quarto document to knit into the specified journal format.

Where does one type a line such as “quarto use template quarto-journals/elsevier" – in the YAML, in a code chunk within the markdown document?

David.

----

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Reply by mcanouil** [reply]
This has to be typed in your terminal.
Quarto is a "command line interface" tool.

**Reply by fortunewalla** [reply]
> Thanks, Mickaël I have visited those web pages, but unable to get my Quarto document to knit into the specified journal format. Where does one type a line such as “quarto use template quarto-journals/elsevier" – in the YAML, in a code chunk within the markdown document? David.

Are you using RStudio to do your present work? Are you using Quarto through RStudio.

BTW RStudio has a built-in terminal: https://support.posit.co/hc/en-us/articles/115010737148-Using-the-RStudio-Terminal-in-the-RStudio-IDE

You can learn how to use this and then try to run quarto CLI commands from there.

**Reply by David-P-Moore** [reply]
Thanks - that was very helpful. The document is rendering perfectly now.

**Reply by fortunewalla** [reply]
Nice. Just mark it as an answer so that this discussion is shown as completed.  👍🏼
